README.md

```diff
@@ -1,339 +1,108 @@
 # nextcloud-dev-docker-compose
 
-Nextcloud development environment using docker-compose
+Hi beginner developer ! 👋
 
-⚠ **DO NOT USE THIS IN PRODUCTION** Various settings in this setup are considered insecure and default passwords and secrets are used all over the place
+This project allows you to get started a nextcloud server and develop your first app !
```

Comment:
```suggestion
This project allows you to start developing a Nextcloud app or contribute to Nextcloud server.
```

Reply:
I could not accept your commit suggestion from UI :/

I apply your request in this commit : https://github.com/juliushaertl/nextcloud-docker-dev/pull/87/commits/201da9717829a8645c02f02619e542d54ca0b154